### ISSUE TITLE
Error: Decimal division by 0

### ISSUE THREAD

**Issue by jessedobbelaere** [infracost/infracost#155]
I'm running the `infracost/infracost:latest` Docker image successfully in a pipeline since Monday, and today it started failing on this error:

```
Using Docker executor with image infracost/infracost:latest ...
Pulling docker image infracost/infracost:latest ...
Using docker image sha256:5ef44616f28a3432db77723bb2dd1f84e367b2e06157bd2ba628f4e4cb82c174 for infracost/infracost:latest with digest infracost/infracost@sha256:4fdd4aaac297708b7982d40ff4648aa40c350130fad8dee786c80362a18583e3 ...

...

Running infracost for production_us-east-1.plan.json
An unexpected error occurred
decimal division by 0
goroutine 1 [running]:
runtime/debug.Stack(0x9532a0, 0xc00000e028, 0xc0000beb88)
	/usr/local/go/src/runtime/debug/stack.go:24 +0x9f
main.main.func3(0xc000001500)
	/app/cmd/infracost/main.go:129 +0x2c5
panic(0x83ec80, 0x949730)
	/usr/local/go/src/runtime/panic.go:969 +0x175
github.com/shopspring/decimal.Decimal.QuoRem(0xc0000bf530, 0x0, 0xc0000befe8, 0x7fa500000000, 0x10, 0xc0000ec6c0, 0x90, 0x88, 0x8a2da0)
	/go/pkg/mod/github.com/shopspring/decimal@v1.2.0/decimal.go:500 +0x385
github.com/shopspring/decimal.Decimal.DivRound(0xc0000bf530, 0x0, 0xc0000befe8, 0xc000000000, 0x10, 0x0, 0x2f)
	/go/pkg/mod/github.com/shopspring/decimal@v1.2.0/decimal.go:542 +0x6e
github.com/shopspring/decimal.Decimal.Div(...)
	/go/pkg/mod/github.com/shopspring/decimal@v1.2.0/decimal.go:487
github.com/infracost/infracost/internal/providers/terraform/aws.getInstanceTypeAndCount(0x5, 0xc000153293, 0x22c, 0x0, 0x0, 0x0, 0x293, 0xc0000bf530, 0x0, 0x40d5b0, ...)
	/app/internal/providers/terraform/aws/autoscaling_group.go:239 +0x525
github.com/infracost/infracost/internal/providers/terraform/aws.newMixedInstancesAwsLaunchTemplate(0xc0002619dd, 0x23, 0xc000113180, 0xc000153571, 0x9, 0xc0000bf530, 0x0, 0x5, 0xc000153293, 0x22c, ...)
	/app/internal/providers/terraform/aws/autoscaling_group.go:198 +0x85
github.com/infracost/infracost/internal/providers/terraform/aws.NewAutoscalingGroup(0xc000112f50, 0x0, 0xc000260687)
	/app/internal/providers/terraform/aws/autoscaling_group.go:74 +0x62a
github.com/infracost/infracost/internal/providers/terraform.createResource(0xc000112f50, 0x0, 0xc000260647)
	/app/internal/providers/terraform/parser.go:27 +0x8d
github.com/infracost/infracost/internal/providers/terraform.parsePlanJSON(0xc0001e0000, 0x70067, 0x7fe00, 0x7fe00, 0x0, 0x0, 0x8c8a0e, 0x7)
	/app/internal/providers/terraform/parser.go:60 +0x6eb
github.com/infracost/infracost/internal/providers/terraform.(*terraformProvider).LoadResources(0xc000077600, 0xc0000775c0, 0x0, 0x0, 0x44c805, 0xc0001201a0)
	/app/internal/providers/terraform/provider.go:60 +0x65
main.defaultCmd.func1(0xc0000775c0, 0xc00000d300, 0x0)
	/app/cmd/infracost/default.go:65 +0xa9
github.com/urfave/cli/v2.(*App).RunContext(0xc000001500, 0x95a780, 0xc00001c058, 0xc000010180, 0x3, 0x3, 0x0, 0x0)
	/go/pkg/mod/github.com/urfave/cli/v2@v2.2.0/app.go:315 +0x70d
github.com/urfave/cli/v2.(*App).Run(...)
	/go/pkg/mod/github.com/urfave/cli/v2@v2.2.0/app.go:215
main.main()
	/app/cmd/infracost/main.go:134 +0x45d
Environment:
Infracost -dirty
Terraform v0.12.25
Please copy the above output and create a new issue at https://github.com/infracost/infracost/issues/new
```

I run `infracost` as following, inside the container:

```bash
echo "Running infracost for $TFJSON_FILE"
infracost --tfjson $TFJSON_FILE
```

Any idea what might be breaking it?

**Comment by aliscott** [comment]
@jessedobbelaere thanks, this is now fixed in v0.6.3.
We've also added versioned docker tags, so you can lock down to an Infracost release.

**Comment by jessedobbelaere** [comment]
Awesome 🙌  It works now 👍  Will pin the version
